tests/integration/test_api/test_config/test_max_upload_size/test_max_upload_size.py

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2015-2021, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+from os.path import join, dirname, realpath
+from uuid import uuid4
+
+import pytest
+import requests
+
+from wazuh_testing.tools.configuration import check_apply_test, get_api_conf
```

Comment:
If it is possible, the imports should be optimized so the code is cleaner.

Reply:
Maybe it is due to the IDE used, but in my case with Pycharm, the order of imports when I use `ctrl+O` is the one I commited in the first place :thinking:.